src/actions/error-action-creators.js

```diff
@@ -63,13 +62,15 @@ function handleAxiosError(err = {}, failActionType) {
       }
     }
   } else if (err.request) {
+    console.log('An axios error occured:', err) // eslint-disable-line
```

Comment:
`axios.err` could be a very big object.
Printing it here seems not to be helpful for debugging.
How about adding `err.config` in the return object?
Let the caller decide if to print the log or not?
How do you think?

Reply:
It looks fine. I'll update the error action payload as axios document. https://github.com/axios/axios#handling-errors